src/module/base.js

```diff
@@ -0,0 +1,93 @@
+import ModiFy from './modify.js';
+import Dataa from './data.js';
+import Remv from './remove.js';
+import todoDom from './toDoDom.js';
+
+const formClearing = (data) => {
+  data.value = '';
+};
+
+const creatData = (data, container, form) => {
+  form.addEventListener('click', (element) => {
+    element.preventDefault();
+    todoDom.data(data.value, container);
+    formClearing(data);
+  });
+};
+
+const creatDataByEnter = (data, container, form) => {
+  form.addEventListener('keypress', (element) => {
+    if (element.key === 'Enter') {
+      element.preventDefault();
+      todoDom.data(data.value, container);
+      formClearing(data);
+    }
+  });
+};
+
+const removData = (container) => {
+  container.addEventListener('click', (element) => {
+    if (element.target.classList.contains('remove-btn')) {
+      const removeBtn = element.target;
+      const dataId = removeBtn.parentElement.getAttribute('data-id');
+      const data = container.querySelector(
+        `li[data-id= "${dataId}"]`,
+      );
+      Remv.data(data);
+    }
+  });
+};
+
+const modifydata = (container) => {
+  container.addEventListener('click', (element) => {
+    if (element.target.classList.contains('data-text')) {
+      const texts = element.target;
+      const data = texts.parentElement.parentElement.parentElement;
+      data.classList.add('green-bg');
+    }
+  });
+
+  container.addEventListener('focusout', (element) => {
+    if (element.target.classList.contains('data-text')) {
+      const texts = element.target;
+      const data = texts.parentElement.parentElement.parentElement;
+      ModiFy.modify(data, texts);
```

Comment:
[OPTIONAL] Please invest more time naming variables and functions. You have some typos and also please use camelCase for your functions.For example
`clearAllComplatedTasks` has typo
   ` Remv.data(task)` is not is not clear about what is doing. 👍